libs/react-helpers/src/hooks/use-yesterday.ts

```diff
@@ -10,13 +11,23 @@ export const now = (roundBy = 1) => {
 /**
  * Returns the yesterday's timestamp rounded by given number (in milliseconds; 5 minutes by default)
  */
-export const useYesterday = (roundBy = DEFAULT_ROUND_BY_MS) => {
-  const yesterday = useRef<number>(now(roundBy) - TWENTY_FOUR_HOURS_MS);
+export const useYesterday = (
```

Comment:
chore (blocking): As this one is now more generic than initially designed I'd rename it to `useAgo` and also make it return a function `(roundBy) => number`. Having that I'd remake `useYesterday` and `useFiveDaysAgo` as:
```
export const useYesterday = useAgo(TWENTY_FOUR_HOURS_MS);
export const useFiveDaysAgo = useAgo(FIVE_DAYS_MS);
```